mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 site_name: "" # already part of the logo
 site_description: "Ask A Question Documentation"
-site_url: https://ask-a-question.com//
+site_url: https://docs.ask-a-question.com/
```

Comment:
Seems like we have to set this to be the docs site URL for versioning to work correctly! https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/#publishing-a-new-version